test/e2e/tinkerbell_test.go

```diff
@@ -34,7 +51,7 @@ func TestTinkerbellKubernetes122SkipPowerActions(t *testing.T) {
 	test.PowerOffHardware()
 	test.PXEBootHardware()
 	test.PowerOnHardware()
-	test.CreateCluster(framework.WithForce())
+	test.CreateCluster(framework.WithForce(), framework.WithControlPlaneWaitTimeout("15m"))
```

Comment:
Is this 15m for just a single control plane or for all control planes?

Reply:
Just the initial control plane node. In case of replicas they use the per machine timeout